### DISCUSSION TITLE
Full Node: Synced - Wallet: not synced

### DISCUSSION THREAD

**Discussion by trazyo** [Chia-Network/chia-blockchain#4951] [Support]
Hi,

I got a problem since 3-4 days that my Full Node is Synced, but my wallet status is not synced.
I already updated to version 1.1.5, deleted my .chia folder, but my wallet keeps being stuck on ,, not synced ''

![bug_1](https://user-images.githubusercontent.com/84182040/118261139-74ffd100-b4b3-11eb-90e9-8265ed59905b.png)
![bug_2](https://user-images.githubusercontent.com/84182040/118261142-75986780-b4b3-11eb-83a0-910d496556ab.png)
![bug_4](https://user-images.githubusercontent.com/84182040/118265701-dcb91a80-b4b9-11eb-94cd-1b488386cb01.png)

I would really appreciate some help or ideas that could fix this problem.

**Comment by khutchings85** [comment]
I have the exact same issue.  Just went through a full sync after deleting .chia directory, I have 1 XCH not showing up in my wallet balance.  Wallet shows one connection but only synced to height 94912.

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
